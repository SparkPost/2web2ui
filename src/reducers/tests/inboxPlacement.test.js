import cases from 'jest-in-case';
import inboxPlacementReducer from '../inboxPlacement';

const TEST_CASES = {
  'list seeds pending': {
    type: 'GET_SEEDS_PENDING',
  },
  'list seeds success': {
    type: 'GET_SEEDS_SUCCESS',
  },
  'list seeds fail': {
    type: 'GET_SEEDS_FAIL',
  },
};

cases(
  'Inbox Placement Reducer',
  ({ payload, type, meta, state }) => {
    expect(inboxPlacementReducer(state, { payload, type, meta })).toMatchSnapshot();
  },
  TEST_CASES,
);
