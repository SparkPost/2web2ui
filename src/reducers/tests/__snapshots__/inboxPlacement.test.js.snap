// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inbox Placement Reducer list seeds fail 1`] = `
Object {
  "seeds": Array [],
  "seedsError": undefined,
  "seedsPending": false,
}
`;

exports[`Inbox Placement Reducer list seeds pending 1`] = `
Object {
  "seeds": Array [],
  "seedsError": null,
  "seedsPending": true,
}
`;

exports[`Inbox Placement Reducer list seeds success 1`] = `
Object {
  "seeds": undefined,
  "seedsError": null,
  "seedsPending": false,
}
`;
